GridTariffAPI.lib/Services/TariffPriceCache.cs

```diff
@@ -99,7 +180,7 @@ private void RefreshCache()
         {
             _tariffPriceStructureRoot = _tariffRepository.GetTariffPriceStructure();
             _holidayRoot = _holidayRepository.GetHolidays();
-            _tariffCacheValidUntil = DateTime.UtcNow.AddMinutes(Constants.CacheConsideredInvalidMinutes).Date;
+            _tariffCacheValidUntil = DateTime.UtcNow.AddMinutes(Constants.CacheConsideredInvalidMinutes).Date; // TODO: endre logikk eller navngiving. Virker merkelig å angi cache-levetiden i minutter, sette den til 24*60 og så trunkere til midnatt. Og dette gjelder jo bare prislistedelen av cachen, ikke per-målepunkt-delen.
```

Comment:
Tanken her har vært å refreshe cache for målepunkttariff hvert døgn.

Reply:
Ja, og det er OK. Men navnet CacheConsideredInvalidMinutes tyder på at det kan styres per minutt.

Reply:
Jeg adresserer dette i feature/GLUE-1310 som det jobbes med.